hoomd/md/alchemy/pair.py

```diff
@@ -0,0 +1,367 @@
+                 norm_value: float = 0.0,
+                 mass: float = 1.0,
+                 mu: float = 0.0):
+        super().__init__(force, name, typepair, alpha, mass, mu)
+        self._param_dict.update(dict(norm_value=norm_value))
+
+    @log(default=False, requires_run=True, category='particle')
+    def alchemical_forces(self):
+        """Per particle forces in alchemical alpha space."""
+        return self._cpp_obj._forces * self._cpp_obj.norm_value
+
+
+@_modify_pair_cls_to_alchemical
+class LJGauss(BaseLJGauss, _AlchemicalPairForce):
+    r"""Alchemical Lennard Jones Gauss pair force.
+
+    Args:
+        nlist (`hoomd.md.nlist.NeighborList`): Neighbor list.
+        default_r_cut (float): Default cutoff radius :math:`[\mathrm{length}]`.
+        default_r_on (float): Default turn-on radius :math:`[\mathrm{length}]`.
+        mode (str): Energy shifting/smoothing mode.
+
+    `LJGauss` computes the Lennard-Jones Gauss force on all particles in the
+    simulation state, with additional alchemical degrees of freedom:
+
+    .. math::
+        U(r) = 1\ [\mathrm{energy}] \cdot \left[
+                 \left ( \frac{1\ [\mathrm{length}]}{r} \right)^{12} -
+                 \left ( \frac{2\ [\mathrm{length}]}{r} \right)^{6} \right] -
+            \alpha_{1}\epsilon
+            \exp \left[ - \frac{\left(r - \alpha_{2}r_{0}\right)^{2}}{2
+            (\alpha_{3}\sigma)^{2}} \right],
+
+    where :math:`\alpha_i` are the alchemical degrees of freedom.
+
+    Note:
+        :math:`\alpha_i` not accessed are set to 1.
+
+    Attention:
+        `hoomd.md.alchemy.pair.LJGauss` does not support execution on GPUs.
+
+    Attention:
+        `hoomd.md.alchemy.pair.LJGauss` does not support MPI parallel
+        simulations.
+
+    .. py:attribute:: params
+
+        The potential parameters. The dictionary has the following keys:
+
+        * ``epsilon`` (`float`, **required**) -
+          energy parameter :math:`\varepsilon` :math:`[\mathrm{energy}]`
+        * ``sigma`` (`float`, **required**) -
+          Gaussian width :math:`\sigma` :math:`[\mathrm{length}]`
+        * ``r0`` (`float`, **required**) -
+          Gaussian center :math:`r_0` :math:`[\mathrm{length}]`
+
+        Type: `TypeParameter` [`tuple` [``particle_type``, ``particle_type``],
+        `dict`]
+
+    .. py:attribute:: epsilon
+
+        Store/mapping of alchemical particles for alchemical degrees of freedom
+        for the potential parameter :math:`\epsilon`.
+
+        Type: `TypeParameter` [`tuple` [``particle_type``, ``particle_type``],
+        `AlchemicalDOF`])
+
+    .. py:attribute:: sigma
+
+        Store/mapping of alchemical particles for alchemical degrees of freedom
+        for the potential parameter :math:`\sigma`.
+
+        Type: `TypeParameter` [`tuple` [``particle_type``, ``particle_type``],
+        `AlchemicalDOF`])
+
+    .. py:attribute:: r0
+
+        Store/mapping of alchemical particles for alchemical degrees of freedom
```

Comment:
The phrasing "store/mapping of alchemical particles for alchemical degrees of freedom..." here bothers me, but I can't come up with a better description. I think we need to drop the "alchemical particles" syntax altogether.

Reply:
Indeed, I just removed "Store/mapping of alchemical particles for" from each of these.